jenkins/jobs/jobs.yaml

```diff
@@ -175,6 +175,49 @@
       - osci-swift-upload-logs-with-console-plugin:
           upload_source: 'test.log logs'
 
+- job-template:
+    name: 'dsvm-tempest-vgpu'
+    node: '{node}'
+
+    wrappers:
+      - timeout:
+          timeout: 30  # Timeout in *minutes*
+          fail: true  # A job run that exceeds the timeout will cause a failure
+      - timestamps
+
+    builders:
+      #- net-info
+      - shell: |
+          #!/bin/bash -x
+          env
+
+          #If disable ceilometer by force
+          export FORCE_DISABLE_CEILOMETER=""
+          export ZUUL_CHANGES=${ZUUL_CHANGES:-""}
+          export ZUUL_REF=${ZUUL_REF:-""}
+          export ZUUL_CHANGE=${ZUUL_CHANGE:-""}
+          export OPENSTACK_LOGS=/tmp/opentack_logs
+
+          echo "Testing vgpu on xenserver"
+          export VGPU_TEST_LOG_DIR=$WORKSPACE/logs
+          LOG=$WORKSPACE/test.log
+          rm -rf $WORKSPACE
+          mkdir -p $WORKSPACE
+          mkdir -p $VGPU_TEST_LOG_DIR
+          cd $WORKSPACE
+          # git clone https://github.com/citrix-openstack/qa -b master
+          cd $HOME/qa/
+          chmod +x test_vgpu.sh
```

Comment:
can we change the file's mode to be executable in the qa repository?

Reply:
Looks OK, will check if it is works.